dev/io.openliberty.security.jakartasec.3.0.internal_fat.config/build.gradle

```
@@ -85,7 +85,11 @@ autoFVT.doLast {
   
      def rpCopyServers = [
 	'jakartasec-3.0_fat.config.rp.claimsDef',
-	'jakartasec-3.0_fat.config.rp.ELOverride'
+	'jakartasec-3.0_fat.config.rp.ELOverride',
+	'jakartasec-3.0_fat.config.rp.userinfo.jwt.jwt',
+	'jakartasec-3.0_fat.config.rp.userinfo.jwt.opaque',
+	'jakartasec-3.0_fat.config.rp.userinfo.jose.jwt',
+	'jakartasec-3.0_fat.config.rp.userinfo.jose.opaque'
```

```suggestion
	'jakartasec-3.0_fat.config.rp.userinfo.json.jwt',
	'jakartasec-3.0_fat.config.rp.userinfo.json.opaque'
```
The User Info responses are in JSON format, so I'd use "json" in the server names. JOSE is a standard for signing and encryption, which is what the JWT formats would use. Using "jose" to refer to the JSON-formatted stuff would be confusing.

---

not sure why I did that - I really did mean to call them json.  
Will update in my next delivery.